scripts/o3de/o3de/android_support.py

```diff
@@ -183,6 +184,22 @@ def get_android_config(project_path: Path or None) -> command_utils.O3DEConfig:
                                     settings_description_list=SUPPORTED_ANDROID_SETTINGS)
 
 
+
+def deloy_to_android_device(target_android_project_path: pathlib.Path,
+                            build_config:str,
+                            android_sdk_home:pathlib.Path,
+                            org_name,
+                            activity_name):
+    adb_exec_path = android_sdk_home / (f'platform-tools/adb{EXE_EXTENSION}')
+    apk_file = target_android_project_path / f'app/build/outputs/apk/{build_config}/app-{build_config}.apk'
+    
+    has_error = exp.process_command([adb_exec_path, 'install', '-t', '-r', apk_file])
+    if has_error:
+        raise AndroidToolError(f"Installing APK '{apk_file}' has failed for the android device.")
+    
+    has_error = exp.process_command([adb_exec_path, 'shell', 'am', 'start', '-a', 'android.intent.action.MAIN','-n', f'{org_name}/.{activity_name}'])
+    if has_error:
+        raise AndroidToolError(f"Unable to start activity {org_name}/.{activity_name}.")
```

Comment:
Why are we launching the android APK right after deployment? We don't launch exported windows projects for export, we shouldn't it for android.